spec/controllers/portal/client_logins_controller_spec.rb

```diff
@@ -1,9 +1,11 @@
 require "rails_helper"
+include ActiveSupport::Testing::TimeHelpers
```

Comment:
I'd like to see the `include` be lower down, e.g. on line 5, so it's within the `RSpec.describe ... do ... end`.